Frontend/src/locales/en.json

```diff
@@ -52,17 +52,17 @@
   },
   "settings": {
     "changeMail": "Change your E-mail here.",
-    "changeUsername": "Change your username here",
-    "changePassword":"Change your password here",
-    "changeProfilePicture": "change your profile pic here",
+    "changeUsername": "Change your username here.",
+    "changePassword":"Change your password here.",
+    "changeProfilePicture": "Change your profile picture here",
     "deleteQuestion": "Delete your Account?",
-    "deleteAccount": "delete",
+    "deleteAccount": "DELETE",
     "oldPassword": "old password",
     "newPassword": "new password",
-    "deleteSure": "Are you sure?",
+    "deleteSure": "Are you sure you want to delete your account?",
     "deleteHint" : "Deleting your account is permanent and irreversible",
-    "deleteYes": "yes",
-    "deleteNo": "no",
+    "deleteYes": "DELETE",
+    "deleteNo": "BACK",
     "profile": "Profile",
     "advice": "Best Size 500x500px",
```

Comment:
Maximale Bildgröße: 50MB. Wir empfehlen ein quadratischen Bildformat.
Maximum filesize: 50MB. We recommend a square picture format.